server/api/auth/auth.test.ts

```diff
@@ -80,11 +81,7 @@ describe('auth router', () => {
       const oldEmail = randEmail();
       const newEmail = randEmail();
 
-      await User.create({
-        discord_user_id: id,
-        discord_name: name,
-        email: oldEmail,
-      });
+      await createUser();
```

Comment:
aha! so - even though tests pass - i don't think it's necessarily testing the same thing.  we need to create the email using the replaced code's deets, otherwise the first assertion (on `userWithOldEmail` will always be `null` because the user is never created, which also makes the second assertion kinda pointless since it's guaranteed by the mocked discord value